news(release): add 2.3.5 release blog post (#294)
mixxxdj/website: content/news/XXXX-XX-XX-mixxx-2-3-5-released.md

title: Mixxx 2.3.5 released
authors: Nikolaus Einhauser
tags: 2.3, 2.3.5, release announcement
comments: yes
status: draft

Another bugfix release, this soon?

> Yes, unfortunately it turned out [2.3.4 introduced a couple serious
> bugs](https://github.com/mixxxdj/mixxx/issues?q=is%3Aissue+is%3Aclosed+milestone%3A2.3.5+label%3Aregression),
> that required prompt action to fix them and to rectify that sub-par release.

Reviewer: ```suggestion
Yes, unfortunately it turned out that [2.3.4 introduced a couple of serious
bugs](https://github.com/mixxxdj/mixxx/issues?q=is%3Aissue+is%3Aclosed+milestone%3A2.3.5+label%3Aregression)
that required prompt action to fix in order to rectify this subpar release.
```

Slightly less colloquial, also there's no comma before 'that' (in this case)

The most serious issue we fixed was a regression that caused an empty waveform
overview after loading a track. This issue has been fixed in version 2.3.5,
along with several other bug fixes and improvements.

> Other notable changes include requesting microphone and line-in access
> permission on macOS, allowing explicit selection of buffers of 2048 and 4096

Reviewer: ```suggestion
Other notable changes include requesting microphone and line-in access
on macOS, allowing explicit selection of buffers of 2048 and 4096
```

or

```suggestion
Other notable changes include requesting permission for microphone and line-in access
on macOS, allowing explicit selection of buffers of 2048 and 4096
```

> frames/period in the JACK API, making the Beat FX section of the Pioneer DDJ-400

Reviewer: `period` sounds a bit out-of-place, I feel like that should be plural word too. Not super familiar with the domain-specifics though, so not sure how to rephrase this.

Author: IMO "frames per period" is the correct terminology. Just like "frames per second". So I'd keep keep "frames/period" or spell out the "/".

Reviewer: Ah okay, I misread that as an 'or', thanks for clarifying!

Author: Thank you for the valid criticism, I updated the section to say "frames per period" since this is so end-user facing, the confusion could be more common.

more intuitive, and fixing a visual issue with the the VU peak indicators in the
Tango skin.

Overall, version 2.3.5 aims to improve stability and performance while
addressing the issues introduced in the previous release. Users are encouraged
to update to this version as soon as possible to ensure the best possible
experience while using Mixxx.

> Here is the full Changelog if you're interested:

Reviewer: ```suggestion
Here is the full change log if you're interested:
```

* Fix empty waveform overview after loading a track (Mixxx 2.3.4 regression)
  Fixed by [#11333](https://github.com/mixxxdj/mixxx/pull/11333)
  [#11359](https://github.com/mixxxdj/mixxx/pull/11359)
  [#11344](https://github.com/mixxxdj/mixxx/issues/11344)
* Fullscreen: Fix a crash that occurs on Linux after enabling fullsceen and using menu
  shortcuts e.g. Alt-F.
  [#11328](https://github.com/mixxxdj/mixxx/pull/11328)
  [#11320](https://github.com/mixxxdj/mixxx/issues/11320)
* Fullscreen: Rebuild & reconnect menu only on desktops with global menu
  [#11350](https://github.com/mixxxdj/mixxx/pull/11350)
* macOS: Request Microphone and line-in access permission.
  [#11367](https://github.com/mixxxdj/mixxx/pull/11367)
  [#11365](https://github.com/mixxxdj/mixxx/issues/11365)

> * JACK API: Allow to explicit select buffers of 2048 and 4096 frames/period. They are not

Reviewer: My gut feeling tells me that "allow to" can't be used that way. Also it should be "explicitly" since it's used as an adverb. Perhaps

```suggestion
* JACK API: Allow explicitly selecting buffers of 2048 and 4096 frames/period. They are not
```

or maybe

```suggestion
* JACK API: Allow for explicit selection of buffers with 2048 and 4096 frames/period. They are not
```

Author: These have been copy-pasted from the changelog, I'd prefer if the different copies of the changelog didn't diverge.

Reviewer: Sure, being consistent makes sense, so feel free to close this, this was just a minor thing I've noticed

title: Mixxx 2.3.5 released
authors: Nikolaus Einhauser
tags: 2.3, 2.3.5, release announcement
comments: yes
status: draft

Another bugfix release, this soon?

Yes, unfortunately it turned out [2.3.4 introduced a couple serious
bugs](https://github.com/mixxxdj/mixxx/issues?q=is%3Aissue+is%3Aclosed+milestone%3A2.3.5),

> we apologize for that, so this is our attempt to rectify that sub-par release.

Reviewer 2: ```suggestion
Yes, unfortunately it turned out [2.3.4 introduced a couple serious
bugs](https://github.com/mixxxdj/mixxx/issues?q=is%3Aissue+is%3Aclosed+milestone%3A2.3.5+label%3Aregression), 
that require an immediate response to fix them and to rectify that sub-par release.
```

There is no need to apologize, instead we can be proud to react that fast. I think the issue was leak of testing where every user can help. But only very few are taking the risk to use an unreleased version, relying on others that are not here. 

In addition I have filtered the linked buglist for regressions only.

The most serious issues we fixed were a regression that caused an empty waveform
overview after loading a track, and a crash on Linux when enabling fullscreen
and using menu shortcuts. These issues have been fixed in version 2.3.5, along

> with several other bug fixes and improvements.

Reviewer 2: ```suggestion
The most serious issues we fixed is a regression that caused an empty waveform
overview after loading a track. This issues have been fixed in version 2.3.5, along
with several other bug fixes and improvements.
```

The menu crasher was an old issue that was already there long before, that no one has ever notices.

Other notable changes include requesting microphone and line-in access
permission on macOS, allowing explicit selection of buffers of 2048 and 4096
frames/period in the JACK API, making the Beat FX section of the Pioneer DDJ-400

> more intuitive, and removing VU peak indicators from the stacked layout in the

Reviewer 2: ```suggestion
more intuitive, and fixing a visual issue of the VU peak indicators in the
```
